examples/rollbar-spring-webmvc/src/main/java/com/example/springwebmvc/ExampleController.java

```diff
@@ -0,0 +1,54 @@
+package com.example.springwebmvc;
+
+import com.rollbar.notifier.Rollbar;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+
+
+@Controller
+public class ExampleController {
+
+  @Autowired
+  private ApplicationContext context;
```

Comment:
If you want to access to the instance of `Rollbar` declared as a `Bean` you don't need to use the Spring's ApplicationContext. You can inject it here by using the constructor.

```
@Autowired
public ExampleController(Rollbar rollbar) {
   this.rollbar = rollbar;
}
```

So then you don't need to do https://github.com/rollbar/rollbar-java/pull/215/files#diff-8f054f106f9618adfbe1e5c9f3222395R26

Reply:
done